test/scenarios/attestation/input/test-scenarios/testDataExport.yaml

```diff
@@ -7,9 +7,8 @@ testScenarios:
   - description: FakedTestScenario
     steps:
       - exampleFile: ../examples/Create_AttestationProvider.json
```

Comment:
who should parse or valid the file path ? oav ?

Reply:
oav will throw error if it think something wrong. In this case we will not use the swagger test scenario